engine/Components/Sanitizer.py

```diff
@@ -24,19 +24,6 @@ def add_brackets(statement):
 	return new_string
 
 
-# def clean_commas(statement: str) -> str:
-# 	"""
-# 	Removes commas if more than one statement is given.
-#
-# 	:param statement: given statement
-# 	:return: returns a statement with commas removed
-# 	"""
-# 	logger.debug("Calling clean_commas(%s)...", statement)
-#
-# 	logger.debug("Returning without commas: statement=%s", statement.replace(",", ""))
-# 	return statement.replace(",", "")
-
-
 def clean_whitespaces(statement: str) -> str:
```

Comment:
No info statements? Is there anyway to know this is happening?

Reply:
I will add some details pertaining to this.